e2e/testrunner/config.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2023 Sauce Labs Inc. All rights reserved.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+package testrunner
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"gopkg.in/yaml.v3"
+)
+
+type Config struct {
+	Runnable
+	ConfigFile any
```

Comment:
This is quite confusing, ConfigFile should be a file not any.
I think we could add `Config any` to `Runnable`, this would further streamline it.